yarn-project/simulator/src/avm/avm_simulator.test.ts

```diff
@@ -105,7 +112,70 @@ describe('AVM simulator: injected bytecode', () => {
   });
 });
 
+const TIMESTAMP = new Fr(99833);
+
 describe('AVM simulator: transpiled Noir contracts', () => {
+  it('bulk testing', async () => {
```

Comment:
Made some debugging easier so i didn't have to run the bb-prover tests. This is probably good to have here anyway.